app/forms/contact_emails_populator.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+# frozen_string_literal: true
+
+# Populates the nested contact email form
+class ContactEmailsPopulator < ApplicationPopulator
+  # The fragment represents one row of the attached file data from the HTML form
+  def call(form, fragment:, **)
+    item = existing_record(form: form, id: fragment['id'])
+
+    if fragment['_destroy'] == '1'
+      form.contact_emails.delete(item)
+      return skip!
```

Comment:
It looks like these two lines may have caused the coverage drop.

Reply:
Thanks @mjgiarlo! Adjusted test for `contact_emails_populator_spec.rb` and now passes with specific test for removal.